README.md

```diff
@@ -16,3 +16,47 @@ The project and Python dependencies can be installed from source with:
 ```
 pip3 install .
 ```
+
+## Configuration
+
+Many EduLinq tools share a common configuration system.
```

Comment:
Anyone is allowed to use this tool, not just EduLinq.
This sentence should go.

This mindset should permeate all the text you are adding here.
I am going to reduce my number of comments here on this pass because this comment will affect so much.